driver/config/config.go

```diff
@@ -732,3 +748,14 @@ func (p *Config) PasswordPolicyConfig() *PasswordPolicy {
 		IgnoreNetworkErrors: p.p.BoolF(ViperKeyIgnoreNetworkErrors, true),
 	}
 }
+
+func (p *Config) HasherPasswordHashingAlgorithm() string {
+	configValue := p.p.StringF(ViperKeyHasherAlgorithm, DefaultPasswordHashingAlgorithm)
+	switch configValue {
+	case "bcrypt":
+		return configValue
+	case "argon2":
```

Comment:
Switches in go break by default :wink: 
```suggestion
	case "argon2":
		fallthrough
```